R/FilteredData.R

```diff
@@ -795,12 +797,23 @@ FilteredData <- R6::R6Class( # nolint
             active_datanames = active_datanames
           )
 
-          shiny::observeEvent(self$get_filter_state(), {
-            if (length(self$get_filter_state()) == 0) {
+          shiny::observeEvent(private$get_filter_count(), {
+            if (private$get_filter_count() == 0) {
               shinyjs::hide("remove_all_filters")
             } else {
               shinyjs::show("remove_all_filters")
             }
+            shinyjs::show("filter_active_vars_contents")
+            shinyjs::hide("filters_active_count")
+            toggle_icon(session$ns("minimise_filter_overview"), c("fa-angle-right", "fa-angle-down"), TRUE)
+            toggle_title(session$ns("minimise_filter_overview"), c("Restore panel", "Minimise Panel"), TRUE)
+          })
+
+          shiny::observeEvent(input[["minimise_filter_overview"]], {
```

Comment:
please stick to one getElement function 
```suggestion
          shiny::observeEvent(input$minimise_filter_overview, {
```

Reply:
The file was already inconsistent, see lines 873 and 853. Should I change all instances to `$`?

Reply:
Yup, please